tensorboard/secured/nginx_singularity.conf

```diff
@@ -0,0 +1,22 @@
+pid /var/run/nginx.pid;
+
+events { worker_connections 1024; }
+
+http {
+  # Turn off the bloody buffering to temp files
+  proxy_buffering off;
+
+  server {
+    listen 59372;
+
+    error_log /dev/null crit;
+
+    server_name localhost;
+
+    location / {
+      proxy_pass http://localhost:6006;
+      auth_basic "Restricted Remote";
+      auth_basic_user_file .htpasswd;
```

Comment:
isn't `/etc/nginx/.htpasswd` ?

Reply:
It can not be `/etc/nginx/` since we are in singularity and not in docker. In fact, these folders are mounted inside the container as `"$LOCAL_PATH"/.nginx/`, this is why I execute the `sed` command in the beginning of the file `singularity-entrypoint.sh`. Here It could be the same, but since I create the `.htpasswd` in the same folder as the `.conf` file I could use the relative path. :)

Reply:
ok thanks for the clarification

Reply:
Sure! 👍